server/lib/nl/explore/params.py

```diff
@@ -44,6 +44,10 @@ class Params(str, Enum):
   CLIENT = 'client'
   # If set then we don't strip stop-words for mono-variable match.
   INCLUDE_STOP_WORDS = 'includeStopWords'
+  SV_THRESHOLD = 'svThreshold'
```

Comment:
varThreshold?  i.e., var to represent topic and SV

(and if you don't feel like changing this, still nodejs param could be varThreshold to match `vars` in json)

Reply:
done